src/server.ts

```diff
@@ -27,6 +27,29 @@ import ConfigResolver, {
 
 } from 'vscode-config-resolver';
 
+interface IWorkspaceSettings {
+	browsers: string[];
+	ignore: string[];
+	ignoreFiles: string[];
+	messageLevel: string;
+	run: string;
+}
+
+type IBrowsersList = string[];
+
+interface IBrowsersListCache {
```

Comment:
JFYI:

```ts
type IBrowsersListCache = Record<string, IBrowsersLise>;
```

Reply:
Wow! I seriously thought that I know TS quite well! I have not yet heard about `Record`s before -.-

Reply:
I really like this! Corrected! ; )

Reply:
- [ ] Replace `[key: string]: string[]` with `Record<>`